examples/angular/basic/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "basic",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
```

Comment:
Maybe a dumb question, but would using Vite with Angular be better or worse?

Reply:
I think we're wanting to consolidate on Vite as our build tooling for both examples and script packaging - might not be a bad idea to switch both over